pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -126,6 +135,38 @@ type Container struct {
 	NetworkSettings *NetworkSettings
 }
 
+// From https://github.com/moby/moby/blob/8dbd90ec00daa26dc45d7da2431c965dec99e8b4/api/types/container/host_config.go#L391
+// HostConfig the non-portable Config structure of a container.
+type HostConfig struct {
+	ExtraHosts      []string          // List of extra hosts
```

Comment:
can we follow the same order as moby for fields in `HostConifg`? It will be easier to see which fields are missing/require modification/etc.